Lusas_Adapter/Private Helpers/GetFeatureResults.cs

```diff
@@ -45,6 +45,7 @@ public partial class LusasV17Adapter
         private Dictionary<string, double> GetFeatureResults(List<string> components, Dictionary<string, IFResultsComponentSet> resultsSets, IFUnitSet unitSet, int id, string suffix, int resultType = 6)
         {
             Dictionary<string, double> featureResults = new Dictionary<string, double>();
+            bool recordedWarning = false;
```

Comment:
```suggestion
            bool invalidResult = false;
```

I think this naming is clearer.

Reply:
Agreed. Points to one thing instead of all warning messages